<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QK2FL40" failures="3" tests="5" name="com.crowdar.bdd.cukes.TestNGSecuencialRunner" time="9.982" errors="2" timestamp="10 nov. 2023 02:42:39 GMT" skipped="0">
  <testcase classname="com.crowdar.bdd.cukes.TestNGSecuencialRunner" name="runScenario" time="3.396">
    <failure message="The status code are not equals, expected [200] and found [401]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The status code are not equals, expected [200] and found [401]
at org.testng.Assert.fail(Assert.java:96)
at io.lippia.api.lowcode.Engine.validates(Engine.java:97)
at io.lippia.api.lowcode.steps.StepsInCommon.status(StepsInCommon.java:96)
at ✽.the status code should be 200(file:src/test/resources/features/Clockifyn't401.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.crowdar.bdd.cukes.TestNGSecuencialRunner" name="runScenario" time="1.524">
    <error message="No results for path: $[0][&#039;id&#039;]" type="com.jayway.jsonpath.PathNotFoundException">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: No results for path: $[0]['id']
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="com.crowdar.bdd.cukes.TestNGSecuencialRunner" name="runScenario" time="0.986">
    <failure message="The status code are not equals, expected [200] and found [401]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The status code are not equals, expected [200] and found [401]
at org.testng.Assert.fail(Assert.java:96)
at io.lippia.api.lowcode.Engine.validates(Engine.java:97)
at io.lippia.api.lowcode.steps.StepsInCommon.status(StepsInCommon.java:96)
at ✽.the status code should be 200(file:src/test/resources/features/Clockifyn't401.feature:33)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.crowdar.bdd.cukes.TestNGSecuencialRunner" name="runScenario" time="1.562">
    <failure message="The status code are not equals, expected [201] and found [401]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The status code are not equals, expected [201] and found [401]
at org.testng.Assert.fail(Assert.java:96)
at io.lippia.api.lowcode.Engine.validates(Engine.java:97)
at io.lippia.api.lowcode.steps.StepsInCommon.status(StepsInCommon.java:96)
at ✽.the status code should be 201(file:src/test/resources/features/Clockifyn't401.feature:25)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.crowdar.bdd.cukes.TestNGSecuencialRunner" name="runScenario" time="2.514">
    <error message="No results for path: $[0][&#039;id&#039;]" type="com.jayway.jsonpath.PathNotFoundException">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: No results for path: $[0]['id']
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.crowdar.bdd.cukes.TestNGSecuencialRunner -->
